addons/l10n_it_edi_sdicoop/models/account_invoice.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding:utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+import logging
+
+from odoo import fields, models
+
+
+_logger = logging.getLogger(__name__)
+
+DEFAULT_FACTUR_ITALIAN_DATE_FORMAT = '%Y-%m-%d'
+
+
+class AccountMove(models.Model):
+    _inherit = 'account.move'
+
+    l10n_it_edi_transaction = fields.Char()
+    l10n_it_edi_attachment_id = fields.Many2one('ir.attachment')
```

Comment:
See https://github.com/odoo/odoo/pull/67923#discussion_r618447230

I know you don't like that, I don't either. This is a result of account_edi only validating a 'sent' if you return an attachment. Maybe we should allow to return an attachment at any time and validating the 'sent' based on the 'success' key like for cancel. As I heard, it might be useful for spain as well, where there might be no attachment.
To be discussed, will not be changed in stable anyway.